uui-editor/src/implementation/ToolbarButton.tsx

```diff
@@ -11,17 +11,25 @@ interface ToolbarButtonProps extends IHasCX, IHasCaption, IDisableable {
     icon?: Icon;
     iconColor?: 'red' | 'green' | 'amber' | 'blue' | 'gray60';
     editor?: Editor;
+    color?: string;
 }
 
-export const ToolbarButton = React.forwardRef<HTMLButtonElement | HTMLAnchorElement, ToolbarButtonProps>((props, ref) => (
-    <Button
-        onClick={ (e) => {
-            props.onClick(e);
-        } }
-        icon={ props.icon }
-        caption={ props.caption }
-        ref={ ref }
-        cx={ cx(css.toolbarButton, css['color-' + props.iconColor], css[props.isActive ? 'gray90' : 'gray80'], props.cx) }
-        isDisabled={ props.isDisabled }
-    />
-));
+export const ToolbarButton = React.forwardRef<HTMLButtonElement | HTMLAnchorElement, ToolbarButtonProps>((props, ref) => {
+    const rawProps = React.useMemo(() => {
```

Comment:
useMemo is useless here, because we didn't make any calculations inside. Moreover using the useMemo for such cases will be less performance, than without it. I supposed we can just inline it.